packages/editor/src/components/inserter/menu.js

```diff
@@ -366,24 +365,54 @@ export default compose(
 			rootClientId,
 		};
 	} ),
-	withDispatch( ( dispatch, ownProps ) => {
+	withDispatch( ( dispatch, ownProps, { select } ) => {
 		const {
 			__experimentalFetchReusableBlocks: fetchReusableBlocks,
 			showInsertionPoint,
 			hideInsertionPoint,
 		} = dispatch( 'core/editor' );
 
+		// eslint-disable-next-line no-restricted-syntax
```

Comment:
We should have a "Disable reason" if only to clarify what's actually being disabled here, given that `no-restricted-syntax` is an umbrella rule containing many miscellaneous custom rules.